packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -179,6 +169,7 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
                 background: "HighlightText !important",

```

Comment:
Should this whole thing be `highContrastSelection` maybe with `border-color` added on? Or should the border always be a part of this combo?

Reply:
put them in a const called `highContrastSelectedOutline`